### DISCUSSION TITLE
更新之后识别不到应用

### DISCUSSION THREAD

**Discussion by adro-5170** [gkd-kit/gkd#763] [Announcements]
### 日志文件

[log-1730450988646.zip](https://github.com/user-attachments/files/17597917/log-1730450988646.zip)

### BUG描述(文字/截图/视频)

部分应用（如哔哩哔哩，必剪）在gkd中显示未安装
![Screenshot_20241101_165303](https://github.com/user-attachments/assets/58b76ebd-1ac3-45dd-8a9d-430714eb0dde)
![Screenshot_20241101_165318](https://github.com/user-attachments/assets/e7ce3ae5-5791-4030-ac09-e8680abd8417)

### 期望行为(文字/截图/视频)

已安装应用能在gkd里被识别

### 实际行为(文字/截图/视频)

gkd无法识别部分应用

**Comment by lisonge** [comment]
你没有授予应用列表读取权限，这是你自己手机的问题

**Reply by lisonge** [reply]
https://github.com/orgs/gkd-kit/discussions/761#discussioncomment-11117596

**Comment by lisonge** [comment]
另外我也不知道为什么有些设备在更新应用之后总是出现数据错乱/权限错乱的问题

你可以在应用信息界面强行停止，或者卸载重装应该可以解决这个问题
